app/css/buddyup.css

```diff
@@ -116,6 +116,15 @@ html {
     padding: .7rem .9rem;
     border-radius: 6px;
 }
+.overlay {
```

Comment:
Actually, let's not add this and make the overlay a new iframe. That will allow us to display it from any page which will be needed if the user kills the app and never validates it's account.